configuration/driver/icedrv_history.F90

```diff
@@ -68,7 +68,11 @@ subroutine history_write()
          count1(1), count2(2), count3(3), count4(4), & ! cdf start/count arrays
          varid, &                        ! cdf varid
          status, &                       ! cdf status flag
-         iflag                           ! history file attributes
+         iflag, &                        ! history file attributes
+         sec0, &                         ! number of seconds into the day at istep0
+         h0, &                           ! start hour
+         m0, &                           ! start minute
+         s0                              ! start second
```

Comment:
Are these actually used?

Reply:
Ah good catch. Those are not actually used. They were used in the original way that I had fixed this. But since we've decided that the start point will always be January 1 00:00:00 and we just use the time variable they are not needed. I'll remove them in the NetCDF restart branch.